app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name):
+        self.name = name
+        self.health = 100
+        self.hidden = False
+        self.alive.append(self)
+
+    def __repr__(self):
+        return f"{{Name: {self.name}, Health:" \
+               f" {self.health}, Hidden: {self.hidden}}}"
+
+
+class Herbivore(Animal):
+    def hide(self):
+        if not self.hidden:
+            self.hidden = True
+        else:
+            self.hidden = False
```

Comment:
you can do it in one line without any condition

Reply:
Yes, it`s really ugly, thanks. Changes were provided.